lib/livebook_web/live/session_live/shortcuts_component.ex

```diff
@@ -4,13 +4,24 @@ defmodule LivebookWeb.SessionLive.ShortcutsComponent do
   @shortcuts %{
     insert_mode: [
       %{seq: ["esc"], desc: "Switch back to navigation mode"},
-      %{seq: ["ctrl", "↵"], press_all: true, desc: "Evaluate cell and stay in insert mode"},
+      %{
+        seq: ["ctrl", "↵"],
+        seq_mac: ["⌘", "↵"],
+        press_all: true,
+        desc: "Evaluate cell and stay in insert mode"
+      },
       %{seq: ["tab"], desc: "Autocomplete expression when applicable"},
       %{
         seq: ["ctrl", "␣"],
         press_all: true,
-        transform_for_mac: false,
         desc: "Show completion list, use twice for details"
+      },
+      %{
+        seq: ["ctrl", "shift", "i"],
+        seq_mac: ["⇧", "⌥", "f"],
```

Comment:
@josevalim please verify that this works as expected for you and if you can undo the formatting editl :)